### PR TITLE
LPS-123637 Styled New External Video Shortcut Preview

### PR DESCRIPTION
**Pull request by boton**
New look:

image

### CODE REVIEW

**Comment by boton** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-external-video/src/main/resources/META-INF/resources/js/ExternalVideoPreview.js [hunk 1]
```diff
@@ -92,28 +92,32 @@ const ExternalVideoPreview = ({
...
-				) : (
-					<div className="file-picker-preview-video">
-						<div className="file-picker-preview-video-placeholder">
-							{loading ? (
-								<ClayLoadingIndicator />
-							) : (
-								<ClayIcon symbol="video" />
-							)}
+				<div className="file-picker-preview-video">
+					{HTML ? (
```
Comment (ambrinchaudhary) [inline modules/apps/document-library/document-library-external-video/src/main/resources/META-INF/resources/js/ExternalVideoPreview.js]: Can't we change this variable name to something more clear? `previewHtml`, `videoHtml`, ... 
Also, notice that we should use lowerCase

**Review by ambrinchaudhary — COMMENTED** [review]
I haven't tested it, but looks good!

File: modules/apps/document-library/document-library-external-video/src/main/resources/META-INF/resources/js/ExternalVideoPreview.js [hunk 2]
```diff
@@ -92,28 +92,32 @@ const ExternalVideoPreview = ({
...
-					/>
-				) : (
-					<div className="file-picker-preview-video">
-						<div className="file-picker-preview-video-placeholder">
-							{loading ? (
-								<ClayLoadingIndicator />
-							) : (
-								<ClayIcon symbol="video" />
-							)}
+				<div className="file-picker-preview-video">
```
[the remark this answers is not in the document]
Comment (boton) [reply modules/apps/document-library/document-library-external-video/src/main/resources/META-INF/resources/js/ExternalVideoPreview.js]: Yep you are right, probably better `file-picker-preview-video-aspect-ratio` and avoid container

**Comment by boton** [comment]
ci:forward

**Comment by ambrinchaudhary** [comment]
ci:forward

**Pull request closed without merging**